README.md

```diff
@@ -201,3 +241,50 @@ To write your own test, see [Writing Tests].
 [Addon Testing Guide]:/docs/Addons.md
 [Grafana dashboards]:https://grafana.datahub.redhat.com/dashboard/db/osd-health-metrics?orgId=1
 [Writing Tests]:/docs/Writing-Tests.md
+ 
+ 
+## osde2ectl
+ 
+The `osde2ectl` command is a cluster provisioning and management tool that is complementary to osde2e. The sub-commands can be viewed using the --help parameter.
+ 
+The ENV variables and config files used for osde2e can be used for osde2ectl as well as it shares the same codebase. The OCM token will also have to be exported before running osde2ectl.
+ 
+Key features include:-
+ 
+1. Cluster creation:
```

Comment:
I'm not a fan of how this gets rendered with the numbered list.

Instead, maybe `###` subheadings for each of the features.

Instead of doing `Example -->` and the code block, I'd also do something like:

**Example:** 
`./osde2ectl create --configs prod --number-of-clusters 4 --batch-size 2 --seconds-between-batches 40`